--- libchipcard-0.9.1/configure.in.orig	Tue Jun 24 11:51:50 2003
+++ libchipcard-0.9.1/configure.in	Fri Jun 27 16:26:56 2003
@@ -90,8 +90,8 @@
 dnl system dependant settings
 case "$OS_TYPE" in
     posix)
-        CHIPCARDC_CFGFILE="`eval echo $prefix/etc`/chipcardc.conf"
-        CHIPCARDD_CFGFILE="`eval echo $prefix/etc`/chipcardd.conf"
+        CHIPCARDC_CFGFILE="/etc/chipcardc.conf"
+        CHIPCARDD_CFGFILE="/etc/chipcardd.conf"
         CHIPCARD_COMMANDS="`eval echo $prefix/share`/libchipcard/commands"
         CHIPCARD_DRIVERS="`eval echo $prefix/share`/libchipcard/drivers"
         CHIPCARDD_PIDDIR="`eval echo $prefix/var`/run"
