--- libchipcard-4.0.0/configure.ac.orig	2007-11-21 21:00:38.000000000 +0100
+++ libchipcard-4.0.0/configure.ac	2007-12-18 08:02:27.005626807 +0100
@@ -513,10 +513,10 @@
 AC_MSG_CHECKING(if symbol visibility is supported)
 visibility_cflags=""
 case `basename $CC` in 
-  gcc*)
+  *-gcc*|gcc*)
      dnl Determine version number (watch out: the outside square
      dnl parenteses are vitally important here!)
-     gccversion=["` ${CC} --version | head -1 | ${SED} 's/.*\([0-9]\.[0-9]\.[0-9]*\).*/\1/' `"]
+     gccversion=["` ${CC} -dumpversion `"]
      case $gccversion in
        4.*)
 	 # Activate -fvisibility=hidden only for explicit 4.x gcc
